app/controllers/concerns/auth.rb

```diff
@@ -0,0 +1,148 @@
+# Controller concern for authentication methods
```

Comment:
I know it's just copy & past code, but it would be nice to do some cleanup if we touch it already - maybe in its own commit by just introducing the concerns